uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/document-list_test.js

```diff
@@ -8,21 +8,23 @@ define([
   'use strict';
 
   describe('DocumentListCtrl', function () {
-    var $controller, $rootScope, DocumentService, $scope, $q, deferred;
+    var $controller, $rootScope, DocumentService, $scope;
 
     beforeEach(module('civitasks.appDashboard'));
-    beforeEach(inject(function (_$controller_, _$rootScope_, _DocumentService_, _$q_) {
+    beforeEach(inject(function (_$controller_, _$rootScope_, _DocumentService_) {
       $controller = _$controller_;
       $rootScope = _$rootScope_;
       $scope = $rootScope.$new();
       DocumentService = _DocumentService_;
-      $q = _$q_;
-      deferred = $q.defer();
     }));
 
+    beforeEach(function (){
+      spyOn(DocumentService, 'get').and.callFake(fakePromise([]));
```

Comment:
`spyOn(DocumentService, 'get').and.returnValue($q.resolve([]));`
`spyOn(DocumentService, 'cacheContactsAndAssignments').and.returnValue($q.resolve([]));`